docs/Examples/Middleware/Main.purs

```diff
@@ -37,7 +37,6 @@ pathMiddleware ::
   HTTPure.Request ->
   HTTPure.ResponseM
 pathMiddleware _ { path: [ "middleware" ] } = HTTPure.ok "Middleware!"
-
 pathMiddleware router request = router request
```

Comment:
`purs-tidy` doesn't insert newlines in equational-style declarations, which I think is the preferred way to write them so there's no confusion about whether it's a new declaration or not.

Reply:
yeah, I never really liked these newlines either